### DISCUSSION TITLE
Release instead of Tag

### DISCUSSION THREAD

**Discussion by tedt10e** [bensheldon/good_job#322] [General]
Hello Ben,

Is it possible to use Release instead of Tag when you release a new version?

Github did not send any notification to the subscriber for a new tag. 
It will only send the notification if we use the new release.

Many Thanks 🙏

**Comment by bensheldon** [comment]
@tedhexaflow thanks for flagging this! I didn't know till now the difference or how Github treats it with notifications.

I'll update the release script to include a step to generate a Release from the tag.

**Reply by bensheldon** [reply]
@tedhexaflow btw, any information you want included in the release? Simplest thing for me is to just link to the Changelog, though I could imagine that if there is any specific Upgrade instructions I would include those too.

**Reply by tedt10e** [reply]
Thanks Ben, I think that’s good enough.
